.goreleaser.yml

```diff
@@ -0,0 +1,46 @@
+builds:
+- id: stanza
+  binary: stanza_{{ .Os }}_{{ .Arch }}
+  main: ./cmd/stanza
+  env:
+  - CGO_ENABLED=0
+  flags:
+    -tags=timetzdata
```

Comment:
We've been bundling tzdata since https://github.com/observIQ/stanza/pull/513 was merged; I see that that PR is pretty old, so before that got merged. I'll make a follow-up PR for this.